astro/src/content/json/themes/templates.json

```diff
@@ -1206,6 +1206,80 @@
     "description": "This page is used after the user has successfully updated their password, or reset it. This page should instruct the user that their password was updated and that they need to login again.",
     "path": "/password/complete"
   },
+  {
+    "displayName": "Phone number verification required",
+    "fieldName": "phoneVerificationRequired",
+    "description": "This page is rendered when a user is required to verify their phone number prior to being allowed to proceed with login. This occurs when `Unverified behavior` is set to `Gated` in identities/phone verification settings on the Tenant.",
+    "path": "/phone/verification-required",
+    "version": "1.99.0",
+    "variables": [
+      {
+        "name": "collectVerificationCode",
+        "type": "Boolean",
+        "description": "When `true`, a form input is displayed to allow a user to enter the verification code. This occurs when `Verification strategy` is set to `FormField` under identities/phone verification settings on the Tenant."
+      },
+      {
+        "name": "phoneNumber",
+        "type": "String",
+        "description": "The current value of the user's phone number. This may be useful to indicate to the user which phone number was sent a verification code."
+      },
+      {
+        "name": "showCaptcha",
+        "type": "Boolean",
+        "description": "A boolean that controls whether or not to include captcha scripts and show the captcha challenge (or message for invisible captcha).",
+        "since": "1.30.0"
```

Comment:
I think remove this because the field has been available in this template since the beginning.